account_statement_import/__manifest__.py

```diff
@@ -1,12 +1,12 @@
-# Copyright 2004-2020 Odoo S.A.
-# Copyright 2020 Akretion France (http://www.akretion.com/)
+# Copyright 2004-2022 Odoo S.A.
+# Copyright 2022 Akretion France (http://www.akretion.com/)
```

Comment:
Are you sure you want to change the years of the copyright claims? It does not seem entirely correct to me.

Reply:
I'm not sure about it. If you know how it should be handled I can skip it.

Reply:
@StefanRijnhart is it ok now?